src/js/controller/model.js

```diff
@@ -185,7 +186,9 @@ const Model = function() {
         if (autoStart !== undefined) {
             this.set('autostart', autoStart);
         }
-        this.set('playOnViewable', this.get('autostart'));
+
+        const autoStartOnMobile = OS.mobile && this.get('autostart');
```

Comment:
```
autoStart = autoStart === undefined ? this.get('autostart') : autoStart;
this.set('playOnViewable', autoStartOnMobile || autoStart === 'viewable');
```

Reply:
or rather
```
if (autoStart === undefined) {
       autoStart = this.get('autostart');
} else {
       this.set('autostart', autoStart);
}
const autoStartOnMobile = OS.mobile && autoStart;
this.set('playOnViewable', autoStartOnMobile || autoStart === 'viewable');
```

Reply:
What does this do?

(`undefined !== 'viewable'`)